public/js/controllers/charts_controller.js

```diff
@@ -254,7 +265,8 @@ export default class extends Controller {
     var gOptions = {
       rollPeriod: this.rollPeriod,
       zoomCallback: null,
-      drawCallback: null
+      drawCallback: null,
+      logscale: this.settings.scale === 'log' ? true : false
```

Comment:
This is where eslint complained during npm run build.

Reply:
Whoops. Thought I pushed this yesterday.